### PR TITLE
Fon flow fix

### PR DESCRIPTION
**Pull request by nsakande**

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/lms/substitutions.conf [hunk 1]
```diff
@@ -134,11 +134,11 @@
   },
   "delay":{
     "weeks":{
-       "one": 604800,
-       "two": 1209600,
-       "three": 1814400,
-       "four": 2419200,
-       "six": 3628800
+       "one": 60,
```
Comment (LKapanad) [inline pepper-apis/studybuilder-cli/studies/lms/substitutions.conf]: I believe this changes are for testing purposes and shouldn't be in the repository

**Review by LKapanad — APPROVED** [review]

File: pepper-apis/studybuilder-cli/studies/lms/substitutions.conf [hunk 2]
```diff
@@ -134,11 +134,11 @@
   },
   "delay":{
     "weeks":{
-       "one": 604800,
-       "two": 1209600,
-       "three": 1814400,
-       "four": 2419200,
-       "six": 3628800
+      "one": 604800,
```
Comment (leonidrakitin) [inline pepper-apis/studybuilder-cli/studies/lms/substitutions.conf:137]: these changes make no sense. Can you just rollback those changes?

**Comment by MocanaAtBroad** [comment]
Can you add the JIRA ticket number to Pull request title and make sure that when merging to develop comment includes the jira ticket number? We are now enforcing Squash and Merge, so only one commit will be merged into develop.

**Comment by Trimax** [comment]
https://github.com/broadinstitute/ddp-study-server/pull/1230

**Pull request closed without merging**